var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -529,3 +529,8 @@ def cmake_args(self):
 
     def setup_run_environment(self, env):
         env.set('DEAL_II_DIR', self.prefix)
+
+    def setup_environment(self, spack_env, run_env):
+        spec = self.spec
+        if '+mpi' in spec:
```

Comment:
I suppose that this should only be done if `+cuda` is in spec as well, i.e.
`if '+cuda' in spec and '+mpi' in spec:`

Reply:
I updated this to unset the CUDAHOSTCXX variable, which seems like a less invasive change for this package (and still works).